<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>30</y>
      <width>461</width>
      <height>411</height>
     </rect>
    </property>
    <widget class="QGroupBox" name="groupBox">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>350</y>
       <width>441</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 253, 192);</string>
     </property>
     <property name="title">
      <string>Actions:</string>
     </property>
     <widget class="QPushButton" name="pushButtonNew">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>80</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 170, 255);</string>
      </property>
      <property name="text">
       <string>New</string>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>images/ic_new.png</normaloff>images/ic_new.png</iconset>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButtonInsert">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>20</y>
        <width>80</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 170, 255);</string>
      </property>
      <property name="text">
       <string>Insert</string>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>images/ic_save.png</normaloff>images/ic_save.png</iconset>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButtonUpdate">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>20</y>
        <width>80</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 170, 255);</string>
      </property>
      <property name="text">
       <string>Update</string>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>images/ic_update.png</normaloff>images/ic_update.png</iconset>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButtonRemove">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>20</y>
        <width>80</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 170, 255);</string>
      </property>
      <property name="text">
       <string>Remove</string>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>images/ic_delete.png</normaloff>images/ic_delete.png</iconset>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>180</y>
       <width>441</width>
       <height>161</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 170, 255);</string>
     </property>
     <property name="title">
      <string>Student Details:</string>
     </property>
     <widget class="QLineEdit" name="lineEditID">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>30</y>
        <width>61</width>
        <height>19</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 127);</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>40</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>ID:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>40</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Code:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEditCode">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>50</y>
        <width>141</width>
        <height>19</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 127);</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEditName">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>70</y>
        <width>141</width>
        <height>19</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 127);</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>70</y>
        <width>45</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Name:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEditAge">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>90</y>
        <width>141</width>
        <height>19</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 127);</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>90</y>
        <width>40</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Age:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEditIntro">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>110</y>
        <width>141</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 127);</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>110</y>
        <width>40</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Intro:</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButtonAvatar">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>130</y>
        <width>71</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Avatar</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButtonRemoveAvatar">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>130</y>
        <width>111</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Remove Avatar</string>
      </property>
     </widget>
     <widget class="QLabel" name="labelAvatar">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>20</y>
        <width>181</width>
        <height>101</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>images/ic_no_avatar.png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>0</y>
       <width>441</width>
       <height>171</height>
      </rect>
     </property>
     <property name="title">
      <string>List of Students:</string>
     </property>
     <widget class="QTableWidget" name="tableWidgetStudent">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>421</width>
        <height>131</height>
       </rect>
      </property>
      <column>
       <property name="text">
        <string>ID</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Code</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Name</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Age</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Intro</string>
       </property>
      </column>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
